### ISSUE TITLE
Add far.ico to distrib

### ISSUE THREAD

**Issue by ctapmex** [FarGroup/FarManager#473]
### Description of the new feature or improvement

I suggest adding "far.ico" to the distribution, so that it can be specified when configuring Windows Terminal. If there are no objections, then I can make a PR for it.

**Comment by alabuzhev** [comment]
> so that it can be specified when configuring Windows Terminal

Have you tried it? I did and I'm [not happy with the result](https://github.com/microsoft/terminal/issues/11732).
It seems that WT (or likely modern Windows UI in general) has issues with the ico support.

Looking at C:\Program Files\WindowsApps\Microsoft.WindowsTerminalPreview_1.12.2931.0_x64__8wekyb3d8bbwe\ProfileIcons, the recommended approach seems to be separate PNGs for different scale factors.

**Comment by ctapmex** [comment]
Yes, I did. But I didn't look closely like you. The current version suits me.
